interfaces/HO-Portal/src/app/users/users-table/users-table.component.ts

```diff
@@ -0,0 +1,86 @@
+import { CommonModule } from '@angular/common';
+import { Component, Input, OnInit } from '@angular/core';
+import { IonicModule } from '@ionic/angular';
+import { TranslateModule } from '@ngx-translate/core';
+import { DateFormat } from 'src/app/enums/date-format.enum';
+import { TableData } from 'src/app/models/user.model';
+import { ProgramsServiceApiService } from 'src/app/services/programs-service-api.service';
+import { SharedModule } from 'src/app/shared/shared.module';
+
+enum StatusName {
+  active = 'Active',
+  inactive = 'Inactive',
+}
+
+enum UserType {
+  admin = 'Admin',
+  regular = 'Regular',
+}
```

Comment:
These string-values are not "data", right? They are to display only, so they need to be translatable.

So they need to be contained in the `en.json`-file.

To 'signal' that a special string is for "data-purposes" only, I would make them all lowercase otherwise.
Or just use an `enum` without any string-values. (TypeScript will use just numbers instead, but that won't matter if it doesn't have to match any database value.)